apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -70,6 +86,16 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
     this.setState(change);
   };
 
+  handleRoleChange = change => {
+    this.setState({professionalRole: change.target.value});
+  };
+
+  handleMultiSelect = index => {
```

Comment:
nit: can you add "grade bands" into this function name as it only works for grade bands?